wiring/inc/spark_wiring_usartserial.h

```diff
@@ -31,6 +31,16 @@
 #include "usart_hal.h"
 #include "spark_wiring_platform.h"
 
+//Available Serial Configuration for C
```

Comment:
I can move it down, but these constants don't quite match the checks happening in usart_hal.cpp.  I could define those though, such as:
#define SERIAL_STOP_BITS 0b00000011
#define SERIAL_PARITY_BITS 0b00001100
etc.

Is this what you mean?